packages/core/src/modules/connections/ConnectionsModule.ts

```diff
@@ -217,31 +231,11 @@ export class ConnectionsModule {
    * @param connectionId the connection record id
    */
   public async deleteById(connectionId: string) {
-    return this.connectionService.deleteById(connectionId)
-  }
-
-  public async findByKeys({ senderKey, recipientKey }: { senderKey: Key; recipientKey: Key }) {
```

Comment:
Is there any reason why this method is deleted? I assume it does not really have anything to do with the `AgentContext` PR?